### DISCUSSION TITLE
Data entry in Carbon

### DISCUSSION THREAD

**Discussion by inacioiso** [carbon-language/carbon-lang#1859] [Getting started]
Hello everyone, I already program in C++ and I'm starting to study Carbon. I wanted to know how the data entry will be. It's something like the "cin" of C++?

**Comment by emillaine** [comment]
I would assume something like:

```
var text = stdin.ReadLine();
var number = text.ParseInt();
```

**Comment by geoffromer** [comment]
We haven't started on the design of the standard library yet, so we don't know.
